_data/nav.yml

```diff
@@ -42,14 +51,13 @@
     - subtitle: "Program Team"
```

Comment:
This needs to be updated/removed.

Reply:
Removing.  I agree if this is going to be for joining the program team or becoming a reviewer, then it needs a more specific page rather than re-using the cfp page.